src/components/AddItem.js

```diff
@@ -37,7 +35,11 @@ const AddItem = () => {
     setGroceryItem("");
     setDaysToPurchase(null);
 
-    history.push("/List");
+    swal("Item added!", {
+      icon: "success",
+      buttons: false,
+      timer: 500,
+    });
```

Comment:
This is so easy! I love it. I would maybe make the timer like 1500, but we can see what feels best to everyone.